ElectronicObserver/Window/Tools/SortieRecordViewer/GameDataExtensions.cs

```diff
@@ -59,9 +59,7 @@ public static class GameDataExtensions
 			Ammo = shipData.Ammo,
 			Range = shipData.Range,
 			Speed = shipData.Speed,
-			LuckModernized = shipData.Kyouka.Skip(4).FirstOrDefault(),
-			HPMaxModernized = shipData.Kyouka.Skip(5).FirstOrDefault(),
-			ASWModernized = shipData.Kyouka.Skip(6).FirstOrDefault(),
+			Kyouka = [.. shipData.Kyouka],
```

Comment:
there's no need to set kyouka here i think ? Since it's set in the constructor ?

Reply:
yup, forgot to remove it after I added it to the constructor